examples/example_job_submit_api.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+"""
+Introductory example - Job Submit API
+=====================================
+
+This example will show how to submit a job
+using the Flux Python bindings.
+"""
+
+
+import os
+import flux
+from flux.job import JobspecV1
+
+#%%
+# Here we instantiate a flux handle. This will connect to the running flux instance.
+# If you were running this on a cluster with Flux, you'd likely already be able
+# to connect. If you are testing out on your own, you might do flux start --test-size=4
+handle = flux.Flux()
+
+#%%
+# This is a new jobspec, or a recipe for a flux job. You'll notice we are providing a command
+# directly, along with tasks, nodes, and cores per task. You could also provide a script here.
```

Comment:
perhaps could add "if we were doing this on the command line it'd be loosely equivalent to what is generated by flux mini submit --ntasks=4 --nodes=2 --cores-per-task=2 sleep 10"